e2e/integration/ssitests.js

```diff
@@ -28,13 +58,13 @@ context( 'Test for SSI', () => {
         cy.get( ".opc-notification-drawer__header > button" ).click( { force: true } )
     })
 
-    it( 'Test for User Profile name', () => {
-        cy.get( 'slot[name="opc-nav-btn"] > button:last-child' ).click( { force: true } ).then( () => {
+    // it( 'Test for User Profile name', () => {
+    //     cy.get( 'slot[name="opc-nav-btn"] > button:last-child' ).click( { force: true } ).then( () => {
 
-                cy.get( '.opc-menu-drawer__menu-header-title' ).should( 'contain.text', `${ Cypress.env( 'USERNAME' ) } sso-tester` );
-            } );
-        cy.get( ".opc-notification-drawer__header > button" ).click( { force: true } )
-    })
+    //             cy.get( '.opc-menu-drawer__menu-header-title' ).should( 'contain.text', `${ Cypress.env( 'USERNAME' ) } sso-tester` );
+    //         } );
+    //     cy.get( ".opc-notification-drawer__header > button" ).click( { force: true } )
+    // })
```

Comment:
Why is this commented?
If it's no longer valid, then you can remove these lines.

Reply:
fixed this as test case was valid